two-ca-chain.txt

```diff
@@ -0,0 +1,13 @@
+.-----------.
+| priv-CA   |
+'-----------'
+      |
+      v
+.-----------.
+| Sub-CA    |
```

Comment:
Should "subordinate" or "intermediate" be used, and do they have different meanings?